simde/x86/avx512/compress.h

```diff
@@ -0,0 +1,40 @@
+#if !defined(SIMDE_X86_AVX512_COMPRESS_H)
+#define SIMDE_X86_AVX512_COMPRESS_H
+
+#include "types.h"
+
+HEDLEY_DIAGNOSTIC_PUSH
+SIMDE_DISABLE_UNWANTED_DIAGNOSTICS
+SIMDE_BEGIN_DECLS_
+
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m256d
+simde_mm256_mask_compress_pd (simde__m256d src, simde__mmask8 k, simde__m256d a) {
+  #if defined(SIMDE_X86_AVX512VL_NATIVE) && defined(SIMDE_X86_AVX512F_NATIVE)
+    return _mm256_mask_compress_pd(src, k, a);
+  #else
+    simde__m256d_private
+    a_ = simde__m256d_to_private(a),
+    src_ = simde__m256d_to_private(src);
+    size_t j = 0;
+    SIMDE_VECTORIZE
+    for (size_t i = 0; i < (sizeof(a_.f64)/sizeof(a_.f64[0])); i++){
```

Comment:
There should be spaces on both sides of the semi-colons to make it easier to see where one clause ends and another begins, as well as on both sides of the division operator for similar reasons.  There should also be a space between the close-paren and the open-curly-brace.  So:

```c
for (size_t i = 0 ; i < (sizeof(a_.f64) / sizeof(a_.f64[0])) ; i++) {
```

Notice how you can kind of tell what is going on just by the groupings; you don't even have to read the code.  That's what we're after; code should always be as easy to read as we can make it.